packages/nodes-base/credentials/WebflowOAuth2Api.credentials.ts

```diff
@@ -16,6 +16,13 @@ export class WebflowOAuth2Api implements ICredentialType {
 			type: 'hidden',
 			default: 'authorizationCode',
 		},
+		{
+			displayName: 'Legacy',
+			name: 'legacy',
+			type: 'boolean',
+			default: true,
+			description: 'If the legacy API should be used',
```

Comment:
when manually testing with legacy option on I get "
403 - {"message":"OAuthForbidden: You are missing the following scopes - 'cms:read'","code":"missing_scopes","externalReference":null,"details":[]}"

Reply:
That makes sense the legacy option is only for the v1 node as we don't have credential versioning yet.

I will update the wording to make it clearer.